partner_street_number/views/assets.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+  <data>
+
+    <template id="assets_backend" name="partner_street_number backend assets" inherit_id="web.assets_backend">
```

Comment:
I checked it once again locally on fresh odoo 9 installation and everything looks fine.